src/helpers/styler.js

```diff
@@ -31,6 +34,11 @@ export function createDraggedElementFrom(originalElement) {
     window.setTimeout(() => draggedEl.style.transition +=`, ${trs('top')}, ${trs('left')}`,0);
     draggedEl.style.zIndex = '9999';
     draggedEl.style.cursor = 'grabbing';
+
+    if(transformDraggedElement) {
```

Comment:
why here and not where you originally had it (when morphing)? i might be misunderstanding the feature/ usecase

Reply:
^ Check above comment